test/tests/time_steppers/iteration_adaptive/tests

```diff
@@ -50,6 +50,7 @@
     csvdiff = 'adapt_tstep_grow_dtfunc_out.csv'
     requirement = "The system shall support the ability to grow the time step size
     when specifying the values of t and dt in the TimeStepper."
+    recover = false
```

Comment:
Why?

Reply:
Restart is already tested below in `[adapt_tstep_grow_dtfunc_restart]`.